app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -25,149 +31,152 @@
     commercial in venue
     - if commercialable_last_version.item
       = link_to commercialable.name,
-            edit_admin_conference_venue_path(conference_id: Conference.find(version.conference_id).short_title,
+            edit_admin_conference_venue_path(conference_id: conference_short_title,
                                                 id: commercialable.id, anchor: 'commercials-content')
     - else
       = commercialable.name
 
   - when 'Conference'
     = link_to 'commercial',
-            admin_conference_commercials_path(conference_id: Conference.find(version.conference_id).short_title)
+            admin_conference_commercials_path(conference_id: conference_short_title)
 
 - when 'EventsRegistration', 'Comment', 'Vote', 'Event'
   = 'event'
   - object = current_or_last_object_state(version.item_type, version.item_id)
   - event_id = object.try(:id) || object.try(:event_id) || object.commentable_id
   = link_to (current_or_last_object_state('Event', event_id).try(:title) || 'deleted event'),
-          admin_conference_program_event_path(conference_id: Conference.find(version.conference_id).short_title, id: event_id)
+          admin_conference_program_event_path(conference_id: conference_short_title, id: event_id)
 
 - when 'Target'
   = 'target'
   - target = current_or_last_object_state(version.item_type, version.item_id)
-  = link_if_alive version, target.to_s, admin_conference_targets_path(conference_id: Conference.find(version.conference_id).short_title)
+  = link_if_alive version, target.to_s, admin_conference_targets_path(conference_id: conference_short_title)
 
 - when 'EventSchedule'
   - event_schedule = current_or_last_object_state(version.item_type, version.item_id)
   event
   = link_to (current_or_last_object_state('Event', event_schedule.event_id).try(:title) || 'deleted event'),
-           admin_conference_program_event_path(conference_id: Conference.find(version.conference_id).short_title, id: event_schedule.event_id)
+           admin_conference_program_event_path(conference_id: conference_short_title, id: event_schedule.event_id)
   in
   =  link_to "Schedule #{event_schedule.schedule_id}",
-            admin_conference_schedule_path(conference_id: Conference.find(version.conference_id).short_title, id: event_schedule.schedule_id)
+            admin_conference_schedule_path(conference_id: conference_short_title, id: event_schedule.schedule_id)
 
 - when 'Schedule'
   = link_if_alive version, "Schedule #{version.item_id}",
-            admin_conference_schedule_path(conference_id: Conference.find(version.conference_id).short_title, id: version.item_id)
+            admin_conference_schedule_path(conference_id: conference_short_title, id: version.item_id)
 
 - when 'Conference'
   = 'conference'
-  = link_to Conference.find(version.conference_id).title,
-          edit_admin_conference_path(id: Conference.find(version.conference_id).short_title)
+  = link_to Conference.find(version.item_id).title,
```

Comment:
This no longer needs any attention since our definition for @versions in the controller depends on the version.conference_id to correspond to an existing conference (for which the user is organizer).
https://github.com/openSUSE/osem/blob/master/app/controllers/admin/versions_controller.rb#L8